### DISCUSSION TITLE
[Suggestion] [Visual Studio] Add an possibility to change Copilot autocomplete shortcut (Tab by default)

### DISCUSSION THREAD

**Discussion by TuTAH1** [community/community#65025] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

Following [Github community discussion](https://github.com/orgs/community/discussions/65015) and [Stackoverflow question](https://stackoverflow.com/questions/76234921)

Copilot have no shortcuts settings (in `Tools → Options →  Envoriment → Keyboard`) for writing (applying) a proposed suggestion (Tab by default, what [interfering IDE's keybindings](https://github.com/orgs/community/discussions/65015)). Please, add it.

**Comment by fbdesignpro** [comment]
Yes! This SHOULD be easy to add, but would really help my workflow, as TAB is very awkward for me to use given other plug-ins etc.

This is actually the future PREVENTING me from using copilot right now. :/

**Reply by ayim** [reply, marked as answer]
If you go into Tools > Options > IntelliCode > Advanced, you'll see an option to set the autocomplete to accept on right arrow.

**Comment by theisjendal** [comment]
It is the same problem with PyCharm!

**Comment by drewboardman** [comment]
I saw this was fixed in PyCharm, but I don't see a fix in Visual Studio (not VSCode). Is that correct?

**Reply by ayim** [reply]
A similar option exists for Visual Studio but is controlled by the autocomplete provider.
To change this setting, go to Tools > Options > IntelliCode > Advanced, you'll see an option to set the autocomplete to accept on right arrow.

**Reply by drewboardman** [reply]
This is different, right? That seems like it turns off the VS autocomplete.

**Reply by ayim** [reply]
This specific sechanges the commit character from tab to ➡️
The other discussion we had disables the autocomplete until you ask for it
